framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/Pages/Shared/Components/AbpPageSearchBox/Default.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿
+<form method="post" class="searchForm">
```

Comment:
We use `kebab-case` for CSS class names.
Rename to `page-search-form` (`search-form` is too general)